packages/apex-node/src/streaming/streamingClient.ts

```diff
@@ -273,9 +274,12 @@ export class StreamingClient {
     testRunId: string
   ): Promise<ApexTestQueueItem> {
     const queryApexTestQueueItem = `SELECT Id, Status, ApexClassId, TestRunResultId FROM ApexTestQueueItem WHERE ParentJobId = '${testRunId}'`;
-    const result = (await this.conn.tooling.autoFetchQuery(
-      queryApexTestQueueItem
-    )) as ApexTestQueueItem;
```

Comment:
So I need a sanity check on how i did these updates.  The autoFetchQuery method returned a promise wrapped Record that we cast to ApexTestQueueItem.  The new alternative as found in https://github.com/forcedotcom/source-tracking/pull/164/files#diff-1f1b4fd1f7261a8b369b878ad3f48f777a395e22cb45185cdfc7c5624d402b8fL645 returns a type that seems to be promise compatible but is confusing.   (it extends EventEmitter).  All the tests pass but I'm not 100% sure this is the correct way to update these queries.